cnv/cnv_users_guide/cnv-installing-qemu-guest-agent.adoc

```diff
@@ -14,7 +14,11 @@ You can also install and start the QEMU guest agent using the
 *cloud-init*:*Use custom script* field of the wizard when creating either
```

Comment:
The explicit string "*Use custom script*" still appears in the generated doc (at least this is how it appears in the test build preview).

Reply:
Thanks for the review @yossisegev. 

1. I added this verification prereq command to both cnv-installing-qemu-guest-agent.adoc and cnv-viewing-ip-of-vm-vnic. I think it makes sense to make it consistent in both places.
2. Actually the way this sentence reads currently is more correct, as displays is active, whereas displayed is passive. We should always go for active voice if possible.
3. For the "use custom script" verbiage, I really need your verification of my rewording as I don't have access to the wizard. i changed  it to: "You can also install and start the QEMU guest agent by using the *custom script* field in the *cloud-init* section of the wizard when creating either virtual machines or virtual machines templates in the web console.

If this is not correct, just let me know and I will fix it.

I'll hold off merging this PR until you verify all this is adequate.

I appreciate the thorough review!! Thanks,

Bob

Reply:
Link to test build: http://file.bos.redhat.com/bgaydos/112619/cnv/cnv_users_guide/